<!DOCTYPE html>
<html>

<head>
	<!-- Site made with Mobirise Website Builder v4.8.1, https://mobirise.com -->
	<meta charset="UTF-8">
	<meta http-equiv="X-UA-Compatible" content="IE=edge">
	<meta name="viewport" content="width=device-width, initial-scale=1, minimum-scale=1">
	<link rel="shortcut icon" href="assets/images/ChEA3.png" type="image/x-icon">
	<meta name="description" content="ChEA3 Web Tool">
	<title>ChEA3</title>
	<link rel="stylesheet" href="assets/web/assets/mobirise-icons/mobirise-icons.css">
	<link rel="stylesheet" href="assets/tether/tether.min.css">
	<link rel="stylesheet" href="assets/bootstrap/css/bootstrap.min.css">
	<link rel="stylesheet" href="assets/bootstrap/css/bootstrap-grid.min.css">
	<link rel="stylesheet" href="assets/bootstrap/css/bootstrap-reboot.min.css">
	<link rel="stylesheet" href="assets/socicon/css/styles.css">
	<link rel="stylesheet" href="assets/dropdown/css/style.css">
	<link rel="stylesheet" href="assets/theme/css/style.css">
	<link rel="stylesheet" href="assets/mobirise/css/mbr-additional.css" type="text/css">
	<link rel="stylesheet" href="assets/networkd3/network_style.css">
	<link rel="stylesheet" href="assets/datatables/jquery.dataTables.min.css">
	<link rel="stylesheet" href="assets/datatables/buttons.dataTables.min.css">
	<link rel="stylesheet" href="assets/datatables/dataTables.bootstrap4.min.css">
	<link rel="stylesheet" href="assets/tooltips/css_tooltips.css">
	<link rel="stylesheet" href="assets/networkd3/network_options.css">
	<link rel="stylesheet" href="assets/spectrum/spectrum.css">
	<link rel="stylesheet" href="assets/chea-query/pretty-headers.css">
	<link rel="stylesheet" href="//code.jquery.com/ui/1.12.1/themes/base/jquery-ui.css">
	<link rel="stylesheet" href="assets/chea-query/result_scroll.css">
	<link rel="stylesheet" href="assets/chea-query/submit-query.css">
	<link rel="stylesheet" href="assets/chea-query/d3barchart.css">
	<link rel="stylesheet" href="assets/networkd3/netprogressbar.css">
	<link rel="stylesheet" href="assets/styles.css">
	<link rel="stylesheet" type="text/css" href="https://cdnjs.cloudflare.com/ajax/libs/Chart.js/2.8.0/Chart.min.css">
	<link rel="stylesheet" type="text/css" href="assets/chea3home/homepage.css">


</head>

<body style="background:white">


	<!-- Navigation Bar -->
	<section class="menu cid-r0JOqR1tf7" id="menu1-0">
		<nav class="navbar navbar-expand beta-menu navbar-dropdown align-items-center navbar-fixed-top navbar-toggleable-sm"
			style="min-height: 25px; padding: 0px">
			<div class="container py-1">
				<button class="navbar-toggler navbar-toggler-right" type="button" data-toggle="collapse"
					data-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false"
					aria-label="Toggle navigation">
					<div class="hamburger">
						<span></span> <span></span> <span></span> <span></span>
					</div>
				</button>
				<div class="menu-logo">
					<div class="navbar-brand" style="margin-left: 10px; min-height: 1rem">
						<span class="navbar-logo"> <a href="#top"> <img src="assets/images/chea3_logo.png" alt="ChEA3"
									title="" style="height: 1.5rem;">
							</a>
						</span> <span class="navbar-caption-wrap"><a class="navbar-caption text-white display-4"
								href="#top" style="padding-right:0;margin-right:0"> ChEA3&nbsp
							</a><a class="navbar-caption text-white display-4"
								style="padding-left:0;margin-left:0;font-weight:300" href="#top">- ChIP-X Enrichment
								Analysis Version 3</a></span>
					</div>
				</div>
				<div class="collapse navbar-collapse" id="navbarSupportedContent"
					style="margin-right: 10px; padding-right: 0px">
					<ul class="navbar-nav nav-dropdown" data-app-modern-menu="true">
						<li class="nav-item"><a class="nav-link link text-white display-4"
								href="index.html#content4-o"><span
									class="mbri-setting mbr-iconfont mbr-iconfont-btn"></span> About</a>
						</li>
						<li class="nav-item"><a class="nav-link link text-white display-4"
								href="index.html#content4-v"><span
									class="mbri-idea mbr-iconfont mbr-iconfont-btn"></span>Tutorial</a></li>
						<li class="nav-item"><a class="nav-link link text-white display-4"
								href="index.html#content4-z"><span
									class="mbri-laptop mbr-iconfont mbr-iconfont-btn"></span>API</a></li>
						<li class="nav-item"><a class="nav-link link text-white display-4"
								href="https://github.com/MaayanLab/chea3web" target="_blank"><span
									class="mbri-github mbr-iconfont mbr-iconfont-btn"></span>GitHub</a></li>

						<li class="nav-item"><a class="nav-link link text-white display-4"
								href="index.html#content4-13"><span
									class="mbri-save mbr-iconfont mbr-iconfont-btn"></span>Download</a></li>
					</ul>

					<!-- <a id="hitcount"> <span class="mbr-iconfont mbr-iconfont-btn"></span></a> -->

				</div>
			</div>
		</nav>
	</section>


	<!-- Block 1.1: Query Submissions -->
	<section class="mbr-fullscreen" id="homepage" style="background:white;padding-bottom:0">
			
			<div class="container" style="background: white">
			<div class="row" style="background: white">
				<div class="col-8 offset-2" style="background:white">
	
					<div id="tfea-submission">
						<h1 class="mbr-section-title mbr-bold mbr-fonts-style display-5 mt-5 mb-0 pb-0">
							Submit Your Gene Set for Analysis with ChEA3
						</h1>
						<!-- <p class="mbr-text mbr-fonts-style display-7"
						style="margin-bottom: 0; padding-bottom: 2rem">Discover
						transcription factors associated with sets of target genes</p> -->
						<div class="form-container" style="padding: 0; border: ''">
							<div class="media-container-column" data-form-type="formoid">
	
								<form class="mbr-form" action="">
									<div class="form-group" data-for="message">
										<button type="button" class="btn btn-link display-7 px-0 py-2"
											style="color: #28a0c9;" id="example-genelist"
											type="submit">Example gene list</button>
										<textarea class="form-control 3px pt-2" rows="7"
											placeholder="Submit gene list with one gene per row." id="genelist"
											cols=""></textarea>
	
									</div>
	
									<button type="button" class="btn btn-primary" id="submit-genelist" type="submit"
										style="padding: .5rem 1.5rem">Submit</button>
									<br>
									<span class="display-7" id="genecheck" style="font-size:70%">0 symbols entered, 0
										duplicates, 0 valid symbols</span>
	
								</form>
	
							</div>
						</div>
					</div>
	
				</div>
	
			</div> <!-- end row -->
		</div> <!-- end container -->
	
		<div id="loading-screen" class="mbr-overlay d-none">
			<div class="c_spinner"></div>
			<div class="ellipsis"></div>
		</div>
	
	</section>
	
	<!-- Block 1.2: Results Page -->
	<section class="mbr-fullscreen d-none" id="results" style="background:white;">
		<div class="container">
			<a class="btn btn-primary float-left" id="newquerybutton" href="/chea3" style="padding: .5rem 1.5rem">New Query</a>
			<div class="h1 text-center my-4">Browse CHEA3 Results</div>
			<nav class="pt-3">
				<div class="nav nav-tabs nav-justified" id="nav-tab" role="tablist">
					<a class="nav-item nav-link rounded-0 active" id="nav-tables-tab" data-toggle="tab" href="#nav-tables" role="tab"
						aria-controls="nav-tables" aria-selected="true">Tables</a>
					<a class="nav-item nav-link rounded-0" id="nav-network-tab" data-toggle="tab" href="#nav-network" role="tab"
						aria-controls="nav-network" aria-selected="false">Network</a>
					<a class="nav-item nav-link rounded-0" id="nav-clustergram-tab" data-toggle="tab" href="#nav-clustergram" role="tab"
						aria-controls="nav-clustergram" aria-selected="false">Clustergram</a>
				</div>
			</nav>
			<div class="tab-content" id="nav-tabContent">
				
				<!-- Tables Tab -->
				<div class="tab-pane border-top-0 rounded-bottom px-4 py-3 fade show active" id="nav-tables" role="tabpanel" aria-labelledby="nav-tables-tab">
					<div id="results-tables" style="background-color: #333333; min-height: 500px;"></div>
				</div>

				<!-- Network Tab -->
				<div class="tab-pane border-top-0 rounded-bottom px-4 py-3 fade" id="nav-network" role="tabpanel" aria-labelledby="nav-network-tab">

					<!-- Ali Code -->
					<div class="tab-content networkcol" id="pills-tabContent">
						<div class="tab-pane fade show active" id="pills-coexpression" role="tabpanel" aria-labelledby="pills-coexpression-tab">
							<div style="background: white" class="networkcol" id="tfnet">
								<!-- Side network navigation menu -->
								<div id="netoptsidenav" class="sidenav" style="float: top">
									<a href="javascript:void(0)" class="closebtn" onclick="closeNav('netoptsidenav')">&times;</a>
									<a class="nav-link link text-white display-4">
										<span class="mbri-change-style mbr-iconfont"></span>
										Color by:
									</a>
									<div class="form-group" style="width: 80%; padding-left: 20px" id="colorbyoptions">
										<select class="form-control" id="colorby" onchange="recolorAllNodes();setGTExLegendView()">
											<option>Tissue (general)</option>
											<option>Tissue (specific)</option>
											<option>GO Enrichment</option>
											<option>WGCNA modules</option>
											<option>none</option>
										</select>
									</div>
									<a class="nav-link link text-white display-4" id="gtextablelink" href="#gtextable">GTEx tissue table</a>
									<a class="nav-link link text-white display-4" onclick="deleteNetwork();zm=1;drawNetwork();">
										<span class="mbri-drag-n-drop2 mbr-iconfont"> </span>Zoom to fit
									</a>
									<a class="nav-link link text-white display-4" onclick="requestFullScreen('tfnet')">
										<span class="mbri-drag-n-drop mbr-iconfont"></span>
										Fullscreen
									</a>
									<a class="nav-link link text-white display-4" onclick="saveSvg('net_svg','network.svg')">
										<span class="mbri-camera mbr-iconfont"> </span>
										Save SVG
									</a>
									<!-- Show node labels: auto, always, never -->
									<div class="checkbox">
										<a class="nav-link link text-white display-4">
											<span class="mbri-edit2 mbr-iconfont"></span>
											Show node labels:
										</a>
									</div>
									<div class="form-group" style="width: 80%; padding-left: 20px">
										<select class="form-control" id="labelview" onchange="setLabelView()">
											<option>auto</option>
											<option>always</option>
											<option>never</option>
										</select>
									</div>
									<a class="nav-link link text-white display-4">
										<span class="mbr-iconfont">
											<input type="checkbox" id="legend_checkbox" class="checkbox-inline" checked="checked" value="1">
										</span>
										Show Legend
									</a>
								</div>
								<div style="width:100%;float:right;display:inline;">
									<span class="mbr-iconfont display-5" style="cursor: pointer;float:right;font-size:1rem" onclick="openNav('netoptsidenav','250px')">&#9776;
										<h6 class="mbr-iconfont display-5" style="font-size: 1rem; display: inline">Network Options</h6>
									</span>
									<div class="form-group" style="width: 175px; float: right; padding-right: 10px">
										<select class="form-control" id="whichnetwork" onchange="changeNetwork()">
											<option>GTEx TF Network</option>
											<option>TCGA TF Network</option>
											<option>ARCHS4 TF Network</option>
										</select>
									</div>
								</div>
							</div>
						</div> <!-- end coexpression tab -->
						<div class="tab-pane fade" id="pills-coreg" role="tabpanel" aria-labelledby="pills-coreg-tab">
							<svg id="coregsvg" width="960" height="600"></svg>
						</div>
					</div> <!-- end tab-contents div -->

				</div>

				<!-- Clustergrammer Tab -->
				<div class="tab-pane border-top-0 rounded-bottom px-4 py-3 fade" id="nav-clustergram" role="tabpanel" aria-labelledby="nav-clustergram-tab">
					<!-- <img src="clustergrammer.png" class="w-100"> -->
					<iframe id="clustergram-iframe" class="w-100" frameborder="0" height="1000"></iframe> 
				</div>
			</div>
		</div>
	</section>


	<!-- Block 2: About ChEA3 -->
	<section class="mbr-section content4 cid-r0K1l83kKh" style="padding-bottom:0; padding-top:0" id="content4-o">


		<div class="container">
			<div class="media-container-row">
				<div class="title col-12 col-md-8">
					<h2 class="align-center pb-3 mbr-fonts-style display-2">About
						ChEA3</h2>


				</div>
			</div>
		</div>
	</section>

	<section class="mbr-section article content12 cid-r0K1pQ9Oau" id="content12-q">


		<div class="container">
			<div class="media-container-row">
				<div class="mbr-text counter-container col-12 col-md-8 mbr-fonts-style display-7">
					Transcription factors (TFs) are proteins that control gene expression by binding
					and unbinding near coding regions to regulate the transcriptional machinery.
					TF enrichment analysis (TFEA) prioritizes transcription factors based on the overlap
					between given lists of differentially expressed genes, and previously annotated TF
					targets assembled from multiple resources. ChEA3 is a web-based TFEA tool. Hence,
					ChEA3 can aid in identifying the TFs responsible for observed changes in gene expression
					when comparing control and perturbation samples. In the past, we have developed and
					published <a
						href="https://academic.oup.com/bioinformatics/article-lookup/doi/10.1093/bioinformatics/btq466">ChEA</a>
					and <a href="https://link.springer.com/chapter/10.1007/978-3-642-40511-2_30">ChEA2</a> which are
					ChIP-seq enrichment analysis tools made of gene set libraries created from published ChIP-seq data
					extracted
					from multiple sources. ChEA3 builds upon these prior versions of ChEA by including more libraries,
					adding benchmarks, and integrative library analyses. <br><br>For ChEA3 the following TF-target gene
					set
					libraries were assembled: Putative targets as determined by ChIP-seq experiments from
					<a href="https://www.encodeproject.org/">ENCODE</a>, <a
						href="http://pedagogix-tagc.univ-mrs.fr/remap/">ReMap</a>,
					and individual publications; co-expression of TFs with other genes based on processed RNA-seq from
					<a href="https://gtexportal.org/home/">GTEx</a> and <a
						href="https://amp.pharm.mssm.edu/archs4/">ARCHS4</a>;
					co-occurrence of TFs with other genes by examining thousands of gene lists submitted to the tool
					<a href="http://amp.pharm.mssm.edu/Enrichr/">Enrichr</a>; and gene signatures resulting from single
					TF perturbations
					followed by genome-wide gene expression experiments. <br><br>The ChEA3 web application outputs
					enrichment
					results in the form of searchable, sortable data tables for each library and integration method.
					An interactive TF co-expression network visualization, clustered using weighted gene co-expression
					network analysis, places the top enrichment results in the context of the larger human
					transcriptional regulatory network.
					Using various benchmarks, we found that integrating libraries boosts performance of ranking TF
					correctly.
					<br><br>
					<strong>Transcription Factor Target Enrichment Analysis</strong> -
					The goal of ChEA3 is to predict transcription factors (TFs) associated with user-input sets of
					genes.
					Discrete query gene sets are compared to
					<a href="index.html#content4-13">ChEA3 libraries</a> of TF target gene sets assembled from multiple
					orthogonal
					'omics' datasets. The Fisher's Exact Test is used to compare the input gene set to the TF target
					gene sets in
					order to determine which TFs may be most closely associated with the input gene set. The gene set
					background
					used to perform these tests is comprised of all approved <a
						href="https://www.genenames.org/">HGNC</a> gene symbols.
					<br>
					<br>

					<strong>Input</strong> - Users can submit a set of human or mouse gene symbols for transcription
					factor enrichment analysis.
					The input gene set can be derived from a variety of sources. Commonly, the gene set may be a set of
					differentially expressed genes,
					with the goal of forming hypotheses about which TFs might be responsible for the changes in gene
					expression. However,
					input from GWAS studies or whole-genome CRISPR-KO screen hits can be used as input to identify
					common regulators for
					these sets of genes.
					<br>
					<br>

					<strong>Network Visualization</strong> - The ChEA3 application displays a scatter plot of points
					representing all human transcription
					factors based on their co-expression similarity. This TF coexpression network was constructed by
					conducting a <a
						href="https://horvath.genetics.ucla.edu/html/CoexpressionNetwork/Rpackages/WGCNA/">Weighted Gene
						Co-expression Network Analysis (WGCNA)</a> over RNA-seq counts of human TFs over all <a
						href="https://gtexportal.org/home/">GTEx</a> samples.
					The network layout was determined using Allegro Edge-Repulsive Strong Clustering in <a
						href="https://cytoscape.org">Cytoscape</a>. WGCNA TF module summary expression profiles, also
					known as Eigen-genes,
					were correlated with gene expression profiles from all GTEx tissue samples to determine strong
					module-tissue associations. Each WGCNA module may be colored by its most
					highly correlated tissue. Tissues are aggregated into more general tissue types (e.g. brain) and
					more specific tissue types (e.g. cerebellum, frontal cortex):

					<table class="table table-striped" id="gtextable"
						style="width: auto !important; margin-left: auto; margin-right: auto;">
						<thead>
							<tr>
								<th style="text-align:left;"> General Tissue </th>
								<th style="text-align:left;"> Specific Tissue </th>
							</tr>
						</thead>
						<tbody>
							<tr>
								<td style="text-align:left;"> Adipose Tissue </td>
								<td style="text-align:left;"> Adipose - Subcutaneous </td>
							</tr>
							<tr>
								<td style="text-align:left;"> Adrenal Gland </td>
								<td style="text-align:left;"> Adrenal Gland </td>
							</tr>
							<tr>
								<td style="text-align:left;"> Blood </td>
								<td style="text-align:left;"> Whole Blood </td>
							</tr>
							<tr>
								<td style="text-align:left;"> Blood </td>
								<td style="text-align:left;"> Cells - EBV-transformed lymphocytes </td>
							</tr>
							<tr>
								<td style="text-align:left;"> Blood Vessel </td>
								<td style="text-align:left;"> Artery - Tibial </td>
							</tr>
							<tr>
								<td style="text-align:left;"> Brain </td>
								<td style="text-align:left;"> Brain - Cerebellum </td>
							</tr>
							<tr>
								<td style="text-align:left;"> Brain </td>
								<td style="text-align:left;"> Brain - Frontal Cortex (BA9) </td>
							</tr>
							<tr>
								<td style="text-align:left;"> Brain </td>
								<td style="text-align:left;"> Brain - Spinal cord (cervical c-1) </td>
							</tr>
							<tr>
								<td style="text-align:left;"> Colon </td>
								<td style="text-align:left;"> Colon - Sigmoid </td>
							</tr>
							<tr>
								<td style="text-align:left;"> Colon </td>
								<td style="text-align:left;"> Colon - Transverse </td>
							</tr>
							<tr>
								<td style="text-align:left;"> Esophagus </td>
								<td style="text-align:left;"> Esophagus - Muscularis </td>
							</tr>
							<tr>
								<td style="text-align:left;"> Esophagus </td>
								<td style="text-align:left;"> Esophagus - Mucosa </td>
							</tr>
							<tr>
								<td style="text-align:left;"> Heart </td>
								<td style="text-align:left;"> Heart - Atrial Appendage </td>
							</tr>
							<tr>
								<td style="text-align:left;"> Liver </td>
								<td style="text-align:left;"> Liver </td>
							</tr>
							<tr>
								<td style="text-align:left;"> Muscle </td>
								<td style="text-align:left;"> Muscle - Skeletal </td>
							</tr>
							<tr>
								<td style="text-align:left;"> Nerve </td>
								<td style="text-align:left;"> Nerve - Tibial </td>
							</tr>
							<tr>
								<td style="text-align:left;"> Pancreas </td>
								<td style="text-align:left;"> Pancreas </td>
							</tr>
							<tr>
								<td style="text-align:left;"> Pituitary </td>
								<td style="text-align:left;"> Pituitary </td>
							</tr>
							<tr>
								<td style="text-align:left;"> Prostate </td>
								<td style="text-align:left;"> Prostate </td>
							</tr>
							<tr>
								<td style="text-align:left;"> Skin </td>
								<td style="text-align:left;"> Skin - Sun Exposed (Lower leg) </td>
							</tr>
							<tr>
								<td style="text-align:left;"> Skin </td>
								<td style="text-align:left;"> Cells - Transformed fibroblasts </td>
							</tr>
							<tr>
								<td style="text-align:left;"> Testis </td>
								<td style="text-align:left;"> Testis </td>
							</tr>
							<tr>
								<td style="text-align:left;"> Thyroid </td>
								<td style="text-align:left;"> Thyroid </td>
							</tr>
							<tr>
								<td style="text-align:left;"> Uterus </td>
								<td style="text-align:left;"> Uterus </td>
							</tr>
						</tbody>
					</table>

					<br>
					<br>

					<strong>Benchmarking Predictions</strong> - The performance of each library and integration method
					was benchmarked using <a href="assets/tflibs/Single-TF_Perturbations.gmt" download">946 single-TF
						loss-of-function (LOF) and gain-of-function (GOF) experiments</a>
					mined from GEO with microarray and RNA-seq readouts. Gene sets were generated from the
					differentially
					expressed genes of these experiments and submitted to the ChEA3 API. The positive class consists of
					ranks assigned by ChEA3 to perturbed TFs.
					The negative class consists of ranks assigned by ChEA3 to TFs not perturbed in the experiment. The
					areas under the receiver operating characteristic
					(ROC) and precision-recall (PR) curves were used to evaluate how well ChEA3 ranked the true
					perturbed TF. Each library has a different number of TFs covered. To enable comparison, the negative
					class
					was downsampled to the size of the positive class to generate PR curves such that a random
					classifier has an AUC of 0.5. These PR curves were bootstrapped 5,000 times to generate mean PR AUCs
					and composite
					PR curves.
					<br>
					<br>

					<img style="width:95%;position:relative" src="./assets/chea3home/images/Fig1.png">
					<strong> Figure 1. Performance of the ChEA3 libraries and integration techniques in recovering the
						perturbed TFs from 946 TF LOF and GOF experiments. a) ROC AUC and mean PR AUC over 5,000
						bootstrapped PR curves b) ROC curves c) Composite PR curves generated from 5,000 bootstrapped
						curves d) The deviation of the cumulative distribution from uniform of the scaled rankings of
						each perturbed TF in the benchmarking dataset.</strong>
				</div>
			</div>
		</div>
	</section>


	<section class="mbr-section content4 cid-r0K2mj4vt0" id="content4-v">


		<div class="container">
			<div class="media-container-row">
				<div class="title col-12 col-md-8">
					<h2 class="align-center pb-3 mbr-fonts-style display-2">
						Tutorial</h2>
					<h3 class="mbr-section-subtitle align-center mbr-light mbr-fonts-style display-5">
						Discover transcription factors associated with your gene set</h3>
					<!-- <div class="mbr-section-btn align-center py-4">
						<a class="btn btn-black-outline display-4"
							href="index.html#video1-b"><span
							class="mbri-video mbr-iconfont mbr-iconfont-btn"></span>LIVE DEMO</a>
					</div> -->
				</div>
			</div>
		</div>
	</section>

	<section class="mbr-section article content11 cid-r0K2ncv4vg" id="content11-w">


		<div class="container">
			<div class="media-container-row">
				<div class="mbr-text counter-container col-12 col-md-8 mbr-fonts-style display-7">
					<ol>
						<li><strong>Submit Your Query</strong> - Submit a query with one gene symbol per row.
							ChEA3 accepts HGNC-approved gene symbols and will discard probe names, transcript IDs,
							and other unrecognizable IDs that have not been converted to gene symbols. You can
							use the example gene set available as a link above the submission box to gain
							familiarity with the expected format. Once submitting your query, you will be presented
							with the results page. Most gene set queries are processed in less than 30 seconds.
							<img src="./assets/chea3home/images/SubmitQuery.gif" style="width:90j%"></li>

						<li><strong>Navigating Your Results</strong> - Results appear as collapsible tables
							in the results side pane. Clicking on the table header expands the table. Tables may
							be searched by entering terms in the search box, sorted by column by clicking on the
							column header, and copied to the clipboard by clicking on the Copy button. Results
							for the ChEA3 TF target libraries are sorted by the Fisher's Exact Test p-value.
							This provides a ranking of transcription factors whose putative transcriptional targets
							are most closely similar to the query set. Integrated results, which take into account
							results from all libraries, are sorted in ascending order by score. Lower scores indicate
							more relevancy to the transcription factor. The top 100 results are listed in each table.
							The full results may be downloaded in JSON format using the button at the bottom of the
							results
							pane. Full results are also available via the
							<a href="index.html#content4-z">ChEA3 API</a>.
							<img src="./assets/chea3home/images/NavigateResults.gif" style="width:90%"></li>

						<li><strong>Interpreting Your Results</strong> - The ChEA3 transcription factor rankings have
							been evaluated by benchmarking the application using query gene sets generated from
							single-transcription factor perturbation followed by genome-wide expression experiments.
							The performance of ChEA3 in highly ranking the "true" perturbed transcription factor
							is summarized using Receiver Operating Characteristic (ROC) and Precision-Recall (PR)
							curves.
							The area under the curves for each TF library, as well as two methods that integrate library
							results
							(Borda count and a Top Rank method) are summarized above (Table 1). For convenience,
							the results from the benchmarking exercise are used to rank the libraries in the results
							pane.
							Hence, we recommend that users consider the quality of their enrichment results in each
							results table based on this order which represents overall quality rank. <img
								src="./assets/chea3home/images/Benchmarking.gif" style="width:90%"></li>

						<li><strong>Visualizing Your Results</strong> - A transcription factor co-expression
							network is provided to help users visualize their top matching transcription factors
							in the context of the larger human transcriptional regulatory network. Network coloring
							options provide additional information about the tissues the transcription factors may be
							most active.
							Sliders and color pickers on each table enable users to highlight the number and color of
							the matching TFs.
							The network may be zoomed, panned, and saved as an SVG image. There are other options, for
							example, hiding
							the legend, switching to full screen, or showing/hiding node labels. These options can be
							accessed by clicking
							the hamburger near the network title.<img
								src="./assets/chea3home/images/VisualizeResults.gif" style="width:90%"> </li>

					</ol>
					<br>
					<br>
					<strong>Run ChEA3 Locally</strong> - The ChEA3 Docker image is available on <a
						href="https://hub.docker.com/r/maayanlab/chea3">DockerHub</a>. In order to run
					the ChEA3 web application locally, Docker must be installed.
					<br><br>
					Pull the ChEA3 Docker image
					<pre style="width: 100%; overflow-x: scroll; white-space: nowrap">$docker pull maayanlab/chea3</pre>

					Run ChEA3 locally on port 5000
					<pre
						style="width: 100%; overflow-x: scroll; white-space: nowrap">$docker run -p 5000:8080 maayanlab/chea3</pre>

					In your web browser:
					<pre style="width: 100%; overflow-x: scroll; white-space: nowrap">localhost:5000/chea3/</pre>


				</div>
			</div>
		</div>
	</section>


	<!-- <section class="cid-r0JPyXePkb" id="video1-b">


		<figure class="mbr-figure align-center container">
			<div class="video-block" style="width: 66%;"></div>
		</figure>
	</section> -->

	<section class="mbr-section content4 cid-r0K2ViwrYH" style="padding-bottom:0" id="content4-z">


		<div class="container">
			<div class="media-container-row">
				<div class="title col-12 col-md-8">
					<h2 class="align-center pb-3 mbr-fonts-style display-2">API</h2>
					<h3 class="mbr-section-subtitle align-center mbr-light mbr-fonts-style display-5">
						ChEA3 has an API to enable programmatic access for querying the ChEA3 database</h3>

				</div>
			</div>
		</div>


		<section class="mbr-section article content1 cid-r0K4JJbbUf" id="content1-11" style="padding-bottom:0">


			<div class="container">
				<div class="media-container-row">
					<div class="mbr-text col-12 col-md-8 mbr-fonts-style display-7">
						The ChEA3 REST API uses GET requests to transport user submitted gene sets and JSON-formatted
						query results between the ChEA3 server and the user’s script. The user gene set and optional
						additional information are encoded in the GET URL.
						To query ChEA3, the GET URL is constructed by appending to the base URL:
						<pre
							style="width: 100%; overflow-x: scroll; white-space: nowrap">http://amp.pharm.mssm.edu/chea3/api/enrich/</pre>
						For example:
						<pre
							style="width: 100%; overflow-x: scroll; white-space: nowrap">https://amp.pharm.mssm.edu/chea3/api/enrich/FOXO1,MYC,EGR1,RELB,ICAM1,ARIDB4</pre>


					</div>
				</div>
			</div>
		</section>

		<section class="mbr-section article content11 cid-r0K4Kn3FRS" id="content11-12" style="padding-bottom:0px">


			<div class="container">
				<div class="media-container-row">
					<div class="mbr-text counter-container col-12 col-md-8 mbr-fonts-style display-7">
						<ol>
							<li><strong>Command-line Example</strong> - The following returns results from all ChEA3
								libraries in JSON format:
								<pre
									style="width: 100%; overflow-x: scroll; white-space: nowrap">$ curl -H "Accept: application/json" https://amp.pharm.mssm.edu/chea3/api/enrich/FOXO1,MYC,EGR1,RELB,ICAM1,ARIDB4</pre>
								Redirect the JSON-formatted results to a file:
								<pre
									style="width: 100%; overflow-x: scroll; white-space: nowrap">$ curl -H "Accept: application/json" https://amp.pharm.mssm.edu/chea3/api/enrich/FOXO1,MYC,EGR1,RELB,ICAM1,ARIDB4 > results.json</pre>
								</a></li>
							<!-- <li><strong>Python Example</strong> - 
								<pre
								style="width: 100%; overflow-x: scroll; white-space: nowrap"></pre></li> -->
							<li><strong>R Example</strong> -
								<pre style="width: 100%; overflow-x: scroll">
								library(jsonlite)
								library(httr)
								base_url <- "https://amp.pharm.mssm.edu/chea3/api/enrich/"
								gene_set <- c("FOXO1","MYC","EGR1","RELB","ICAM1","ARIDB4")
								#collapse gene set to 'FOXO1,MYC,EGR1,RELB,ICAM1,ARIDB4'
								gene_set <- paste(gene_set,collapse=",")
								call = paste(base_url,gene_set,sep='')
								response = GET(call)
								json = content(response,"text")
								#results as R list of dataframes
								results = fromJSON(json)
								</pre>
							</li>
						</ol>
					</div>
				</div>
			</div>
		</section>
	</section>

	<!-- DOWNLOAD LIBRARIES SECTION -->

	<section class="mbr-section content4 cid-r0K4Mm8LAA" style="padding-bottom:0" id="content4-13">


		<div class="container">
			<div class="media-container-row">
				<div class="title col-12 col-md-8">
					<h2 class="align-center pb-3 mbr-fonts-style display-2">
						Download</h2>


				</div>
			</div>
		</div>
	</section>
	<h3 class="mbr-section-subtitle align-center mbr-light mbr-fonts-style display-5">
		Libraries used for ChEA3 enrichment analysis</h3>


	<section class="features5 cid-r0K1ARyThe" id="features5-r">


		<div class="container">


			<div class="media-container-row">

				<div class="card p-3 col-12 col-md-6 col-lg-3">
					<div class="card-img pb-3">
						<a href="assets/tflibs/ARCHS4_Coexpression.gmt" download><span class="mbri-save mbr-iconfont"
								style="cursor: pointer; display:table; margin:0 auto"></span></a>
					</div>
					<div class="card-box">
						<h4 class="card-title py-3 mbr-fonts-style display-7" style="text-align:center"><a
								href="https://amp.pharm.mssm.edu/archs4/">ARCHS4</a>
							Coexpression</h4>
						<div id="module" class="container">

							<p class="mbr-text mbr-fonts-style display-7 collapse" style="text-align: justify"
								id="collapseARCHS" aria-expanded="false">
								<a href="https://amp.pharm.mssm.edu/archs4/">ARCHS4</a>
								is a massive RNA-seq reprocessing effort containing over 150,000
								human and mouse samples from GEO. The ARCHS4 TF co-expression
								library was generated by randomly sampling 50,000 human RNA-seq
								samples from ARCHS4, building a gene-gene Pearson correlation
								matrix, and then taking the 300 genes with the greatest absolute value
								of the Pearson correlation coefficient with each of 1,618 unique human
								transcription factors for a total of 1,618 gene sets. The library
								encompasses 485,400 putative TF-gene interactions.
							</p>
							<a id="showmoreless" role="button" class="collapsed" data-toggle="collapse"
								href="#collapseARCHS" aria-expanded="false" aria-controls="collapseARCHS"></a>
						</div>
					</div>

				</div>

				<div class="card p-3 col-12 col-md-6 col-lg-3">
					<div class="card-img pb-3">
						<a href="assets/tflibs/GTEx_Coexpression.gmt" download><span class="mbri-save mbr-iconfont"
								style="cursor: pointer; display:table; margin:0 auto"></span></a>
					</div>
					<div class="card-box">
						<h4 class="card-title py-3 mbr-fonts-style display-7" style="text-align:center"><a
								href="https://gtexportal.org/home/">GTEx</a>
							Coexpression</h4>
						<div id="module" class="container">
							<p class="mbr-text mbr-fonts-style display-7 collapse" id="collapseGTEx"
								aria_expanded="false" style="text-align: justify">
								The <a href="https://gtexportal.org/home/">Genotype-Tissue Expression (GTEx)</a>
								project is a public resource dedicated to cataloguing tissue-specific
								gene expression and regulation. Samples were collected from 53 healthy tissue sites
								across almost 1000 individuals,
								for molecular assays including whole-genome sequencing, whole-exome sequencing, and
								RNA-seq.
								All human GTEx RNA-seq samples
								were used to build a gene-gene Pearson correlation matrix across
								all tissues available.
								The 300 genes with the greatest absolute value
								of the Pearson correlation coefficient for each of 1,598 unique human
								transcription were used to build the TF gene set library. The
								library encompasses 479,400 putative TF-gene interactions.
							</p>
							<a id="showmoreless" role="button" class="collapsed" data-toggle="collapse"
								href="#collapseGTEx" aria-expanded="false" aria-controls="collapseGTEx"></a>
						</div>
					</div>
				</div>


				<div class="card p-3 col-12 col-md-6 col-lg-3">
					<div class="card-img pb-3">
						<a href="assets/tflibs/Enrichr_Queries.gmt" download><span class="mbri-save mbr-iconfont"
								style="cursor: pointer; display:table; margin:0 auto"></span></a>
					</div>
					<div class="card-box">
						<h4 class="card-title py-3 mbr-fonts-style display-7" style="text-align:center"><a
								href="http://amp.pharm.mssm.edu/Enrichr/">Enrichr</a>
							Queries</h4>
						<div id="module" class="container">
							<p class="mbr-text mbr-fonts-style display-7 collapse" style="text-align: justify"
								id="enrichrCollapse" aria-expanded="false">
								User-submitted gene lists were copied from the <a
									href="http://amp.pharm.mssm.edu/Enrichr/">Enrichr</a>
								for a total of 1,097,157 unique lists. Enrichr example lists, lists used for internal
								testing, lists with greater than 2000 genes, lists with less than 2 genes, and lists
								from IP addresses that submitted >1000 lists were filtered out which resulted in 293,747
								unique gene lists remaining for co-occurrence analysis.
								For each TF <i>i</i> defined in a census of 1404 TFs, the probability of co-occurrence
								of the transcription factor TF<sub>i</sub> and a gene g<sub>j</sub> P(TF<sub>i</sub> ∩
								g<sub>j</sub>) was computed for all genes in G. This is essentially the count of the
								number of times TF<sub>i</sub> and g<sub>j</sub> appear together in the same list across
								the user-submitted gene lists. The top 300 co-occurring genes for each TF<sub>i</sub>
								were used. to generate the GMT gene lists.
								The library encompasses 419,114
								TF-gene interactions.
							</p>
							<a id="showmoreless" role="button" class="collapsed" data-toggle="collapse"
								href="#enrichrCollapse" aria-expanded="false" aria-controls="enrichrCollapse"></a>
						</div>
					</div>
				</div>


			</div>
		</div>

	</section>

	<section class="features5 cid-r0K1D0ndDf" id="features5-s">


		<div class="container">
			<div class="media-container-row ">

				<div class="card p-3 col-12 col-md-6 col-lg-3">
					<div class="card-img pb-3">
						<div class="center">
							<a href="assets/tflibs/ReMap_ChIP-seq.gmt" download><span class="mbri-save mbr-iconfont"
									style="cursor: pointer; display:table; margin:0 auto"></span></a>
						</div>
					</div>
					<div class="card-box mbr-fonts-style">
						<h4 class="card-title py-3 display-7" style="text-align:center"><a
								href="http://pedagogix-tagc.univ-mrs.fr/remap/">ReMap</a> ChIP-seq</h4>
						<div id="module" class="container">
							<p class="mbr-text mbr-fonts-style display-7 collapse" id="remapCollapse"
								aria-expanded="false" style="text-align: justify">
								The ReMap ChIP-seq library was generated using BED files from the
								<a href="http://pedagogix-tagc.univ-mrs.fr/remap/">ReMap Project</a>. BED files were
								annotated with
								a <a
									href="https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1003342">distance-weighted
									peak-to-gene model</a> and the top 10% of
								transcription start sites were assigned to each of 300 unique TFs
								to generate putative target sets, encompassing 422,025 TF-gene
								interactions.
							</p>
							<a id="showmoreless" role="button" class="collapsed" data-toggle="collapse"
								href="#remapCollapse" aria-expanded="false" aria-controls="remapCollapse"></a>
						</div>
					</div>
				</div>


				<div class="card p-3 col-12 col-md-6 col-lg-3">
					<div class="card-img pb-3">
						<a href="assets/tflibs/ENCODE_ChIP-seq.gmt" download><span class="mbri-save mbr-iconfont"
								style="cursor: pointer; display:table; margin:0 auto"></span></a>
					</div>
					<div class="card-box">
						<h4 class="card-title py-3 mbr-fonts-style display-7" style="text-align:center"><a
								href="https://www.encodeproject.org/">ENCODE</a>
							ChIP-seq</h4>
						<div id="module" class="container">
							<p class="mbr-text mbr-fonts-style display-7 collapse" id="encodeCollapse"
								aria-expanded="false" style="text-align: justify">
								TF target gene sets were generated for the web tool <a
									href="http://amp.pharm.mssm.edu/Enrichr/">Enrichr</a>
								using uniformly reprocessed <a href="https://www.encodeproject.org/">ENCODE</a>
								TF ChIP-seq experiments. Peak calling on alignment files was conducted using the MACs
								software. Peaks
								were then sorted by distance to the transcription start site and 2000 peaks at most were
								retained for each
								experiment. The library encompasses 552 ChIP-seq experiments of 118 unique TFs. The
								library contains 867,121
								TF-gene interactions.
							</p>
							<a id="showmoreless" role="button" class="collapsed" data-toggle="collapse"
								href="#encodeCollapse" aria-expanded="false" aria-controls="encodeCollapse"></a>
						</div>
					</div>
				</div>

				<div class="card p-3 col-12 col-md-6 col-lg-3">
					<div class="card-img pb-3">
						<a href="assets/tflibs/Literature_ChIP-seq.gmt" download><span class="mbri-save mbr-iconfont"
								style="cursor: pointer;  display:table; margin:0 auto"></span></a>
					</div>
					<div class="card-box">
						<h4 class="card-title py-3 mbr-fonts-style display-7" style="text-align:center"><a
								href="https://academic.oup.com/bioinformatics/article/26/19/2438/230298">Literature
								ChIP-seq </a>(ChEA)</h4>
						<div id="module" class="container">
							<p class="mbr-text mbr-fonts-style display-7 collapse" id="collapseLit"
								aria-expanded="false" style="text-align: justify">
								This library is derived from TF ChIP-seq and ChIP-ChIP experiments mined from
								supplemental data in the literature. Previous versions of this
								library were used in <a
									href="https://academic.oup.com/bioinformatics/article/26/19/2438/230298"
									target="_blank">ChEA</a> and <a
									href="https://link.springer.com/chapter/10.1007/978-3-642-40511-2_30"
									target="_blank">ChEA2</a>. The library gene set metadata includes the PubMed ID
								number
								for the publication from which the experiment originated, as well as the species, assay
								and the cell- or tissue-type.
								This library encompasses 463 ChIP-seq and ChIP-ChIP experiments across 282 publications.
								The gene sets
								are associated with 194 unique TFs and contains 623,233 TF-gene interactions.

							</p>
							<a id="showmoreless" role="button" class="collapsed" data-toggle="collapse"
								href="#collapseLit" aria-expanded="false" aria-controls="collapseLit"></a>

						</div>
					</div>
				</div>


			</div>
		</div>
	</section>
	<h3 class="mbr-section-subtitle align-center mbr-light mbr-fonts-style display-5">
		Benchmarking Libraries</h3>

	<section class="features5 cid-r0K1D0ndDf" id="features5-s">


		<div class="container">
			<div class="media-container-row">

				<div class="card p-3 col-12 col-md-6 col-lg-3">
					<div class="card-img pb-3">
						<a href="assets/tflibs/Single-TF_Perturbations.gmt" download><span
								class="mbri-save mbr-iconfonto"
								style="cursor: pointer; display:table; margin:0 aut"></span></a>
					</div>
					<div class="card-box">
						<h4 class="card-title py-3 mbr-fonts-style display-7" style="text-align:center">TF LOF/GOF
							Experiments</h4>
						<div id="module" class="container">
							<p class="mbr-text mbr-fonts-style display-7 collapse" aria-expanded="false"
								id="collapsePert" style="text-align: justify">
								Single-TF perturbation experiments including knockdowns, knockouts, overexpression,
								and chemical inhibition followed by genome-wide microarray profiling were manually
								curated by the crowd for the from <a
									href="https://www.nature.com/articles/ncomms12846">CREEDS</a>.
								An additional 160 single-TF perturbation from RNA-seq experiments were manually curated
								from GEO by
								first programmatically searching <a href="https://amp.pharm.mssm.edu/archs4/">ARCHS4</a>
								metadata for potential studies that contain relevant signatures.
								The top 300 upregulated, and the top 300 downregulated genes were used to compose TF
								perturbation experiment -
								associated gene sets for a total of 786 gene sets associated with 275 unique
								site-specific TFs.
							</p>
							<a id="showmoreless" role="button" class="collapsed" data-toggle="collapse"
								href="#collapsePert" aria-expanded="false" aria-controls="collapsePert"></a>

						</div>
					</div>
				</div>

				<div class="card p-3 col-12 col-md-6 col-lg-3">
					<div class="card-img pb-3">
						<a href="assets/tflibs/Cusanovich_TF_shRNA.gmt" download><span class="mbri-save mbr-iconfont"
								style="cursor: pointer;  display:table; margin:0 auto"></span></a>
					</div>
					<div class="card-box">
						<h4 class="card-title py-3 mbr-fonts-style display-7" style="text-align:center"><a
								href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE50588">TF shRNA
								Experiments</a></h4>
						<div id="module" class="container">
							<p class="mbr-text mbr-fonts-style display-7 collapse" aria-expanded="false"
								id="collapseCus" style="text-align: justify">
								The top 300 upregulated and top 300 downregulated genes from each of 49 unique TF shRNA
								knockdowns by <a
									href="https://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1004226"><i>Cusanovich
										et al.</i></a> were used to generate 49 TF perturbation-associated gene sets.
							</p>
							<a id="showmoreless" role="button" class="collapsed" data-toggle="collapse"
								href="#collapseCus" aria-expanded="false" aria-controls="collapseCus"></a>

						</div>
					</div>
				</div>


			</div>
		</div>
	</section>


	<h3 class="mbr-section-subtitle align-center mbr-light mbr-fonts-style display-5">
		Additional Libraries</h3>
	<section class="features5 cid-r0K1D0ndDf" id="features5-s">

		<div class="container">
			<p>The gmts in this section are included solely as additional sources of information that might be of
				interest to ChEA3 users.
				These sets are not used by the tool in performing over-representation analysis. In the future, some of
				these sets may be validated and
				included in subsequent versions of the ChEA3 tool.</p>
			<div class="media-container-row">

				<div class="card p-3 col-12 col-md-6 col-lg-3">
					<div class="card-img pb-3">
						<a href="assets/tflibs/tf_humap_ppi.gmt" download><span class="mbri-save mbr-iconfont"
								style="cursor: pointer; display:table; margin:0 auto"></span></a>
					</div>
					<div class="card-box">
						<h4 class="card-title py-3 mbr-fonts-style display-7" style="text-align:center"><a
								href="http://proteincomplexes.org">Hu.MAP</a> TF PPIs</h4>
						<div id="module" class="container">
							<p class="mbr-text mbr-fonts-style display-7 collapse" style="text-align: justify"
								aria-expanded="false" id="collapseHuMap">
								<a href="http://proteincomplexes.org">Hu.MAP</a> integrates large-scale affinity
								purification mass-spec datasets. All putative protein interacters with each TF were
								used to generate this TF PPI library.
							</p>
							<a id="showmoreless" role="button" class="collapsed" data-toggle="collapse"
								href="#collapseHuMap" aria-expanded="false" aria-controls="collapseHuMap"></a>

						</div>
					</div>
				</div>



				<div class="card p-3 col-12 col-md-6 col-lg-3">
					<div class="card-img pb-3">
						<a href="assets/tflibs/tf_iref_ppi.gmt" download><span class="mbri-save mbr-iconfont"
								style="cursor: pointer; display:table; margin:0 auto"></span></a>
					</div>
					<div class="card-box">
						<h4 class="card-title py-3 mbr-fonts-style display-7" style="text-align:center"><a
								href="http://wodaklab.org/iRefWeb/">iRefWeb</a> TF PPIs</h4>
						<div id="module" class="container">
							<p class="mbr-text mbr-fonts-style display-7 collapse" aria-expanded="false"
								id="irefCollapse" style="text-align: justify">
								iRefWeb integrates PPIs from several major public PPI databases. PPIs from the <a
									href="https://thebiogrid.org/">BioGRID</a>, <a
									href="http://mips.helmholtz-muenchen.de/proj/ppi/">MPPI</a>,
								<a href="http://mips.helmholtz-muenchen.de/corum/">CORUM</a>,
								and <a href="http://www.hprd.org/">HPRD</a> databases
								are included in this library.
							</p>
							<a id="showmoreless" role="button" class="collapsed" data-toggle="collapse"
								href="#irefCollapse" aria-expanded="false" aria-controls="irefCollapse"></a>

						</div>
					</div>
				</div>



			</div>
		</div>
	</section>

	<section class="accordion1 cid-rnbkpN1hCU" id="accordion1-d"
		style="background-color:#FFFFFF;margin:50px;padding:50px">




		<div class="container">
			<div class="media-container-row">
				<div class="col-12 col-md-8">
					<div class="section-head text-center space30">
						<h2 class="mbr-section-title pb-5 mbr-fonts-style display-2">
							ChEA3 FAQ
						</h2>
					</div>
					<div class="clearfix"></div>
					<div id="bootstrap-accordion_9" class="panel-group accordionStyles accordion" role="tablist"
						aria-multiselectable="true">
						<div class="card">
							<div class="card-header" role="tab" id="headingOne">
								<a role="button" class="panel-title collapsed text-black" data-toggle="collapse"
									data-core="" href="#collapse1_9" aria-expanded="false" aria-controls="collapse1">
									<h4 class="mbr-fonts-style display-7">
										<span class="sign mbr-iconfont mbri-arrow-down inactive"></span>
										Which species are included in ChEA3 and what type of gene ids or symbols are
										accepted?
									</h4>
								</a>
							</div>
							<div id="collapse1_9" class="panel-collapse noScroll collapse " role="tabpanel"
								aria-labelledby="headingOne" data-parent="#bootstrap-accordion_9">
								<div class="panel-body p-4">
									<p class="mbr-fonts-style panel-text display-7">
										The ARCHS4, GTEx, and ReMap ChEA3 libraries were constructed with human samples.
										The Enrichr Queries library was constructed with HGNC-mappable gene symbols. The
										ENCODE and Literature
										ChIP-seq libraries were constructed with both mouse and human samples.
										ChEA3 accepts HGNC-approved gene symbols for the user-submitted query gene set.
										Frequently, these symbols map directly to mouse symbols (ChEA3 is not
										case-sensitive). ChEA3 has been
										validated against a benchmarking set of mouse experiments, which performed
										comparably to a set of human experiments. Therefore, we recommend ChEA3 for use
										with human and mouse sets.
										Note that the mouse gene symbols that do not map directly to HGNC-approved human
										gene symbols will not be considered by the tool.
									</p>
								</div>
							</div>
						</div>
						<div class="card">
							<div class="card-header" role="tab" id="headingTwo">
								<a role="button" class="collapsed panel-title text-black" data-toggle="collapse"
									data-core="" href="#collapse2_9" aria-expanded="false" aria-controls="collapse2">
									<h4 class="mbr-fonts-style display-7">
										<span class="sign mbr-iconfont mbri-arrow-down inactive"></span> Does ChEA3
										indicate the direction of TF activity?
									</h4>
								</a>

							</div>
							<div id="collapse2_9" class="panel-collapse noScroll collapse" role="tabpanel"
								aria-labelledby="headingTwo" data-parent="#bootstrap-accordion_9">
								<div class="panel-body p-4">
									<p class="mbr-fonts-style panel-text display-7">
										...
									</p>
								</div>
							</div>
						</div>
						<div class="card">
							<div class="card-header" role="tab" id="headingThree">
								<a role="button" class="collapsed text-black panel-title" data-toggle="collapse"
									data-core="" href="#collapse3_9" aria-expanded="false" aria-controls="collapse3">
									<h4 class="mbr-fonts-style display-7">
										<span class="sign mbr-iconfont mbri-arrow-down inactive"></span> Should I input
										upregulated genes, downregulated genes, or a query set that is upregulated and
										downregulated genes combined?
									</h4>
								</a>
							</div>
							<div id="collapse3_9" class="panel-collapse noScroll collapse" role="tabpanel"
								aria-labelledby="headingThree" data-parent="#bootstrap-accordion_9">
								<div class="panel-body p-4">
									<p class="mbr-fonts-style panel-text display-7">
										Benchmarking results indicate...</p>
								</div>
							</div>
						</div>
						<div class="card">
							<div class="card-header" role="tab" id="headingFour">
								<a role="button" class="collapsed text-black panel-title" data-toggle="collapse"
									data-core="" href="#collapse4_9" aria-expanded="false" aria-controls="collapse4">
									<h4 class="mbr-fonts-style display-7">
										<span class="sign mbr-iconfont mbri-arrow-down inactive"></span> Which is the
										best results library to use?
									</h4>
								</a>
							</div>
							<div id="collapse4_9" class="panel-collapse noScroll collapse" role="tabpanel"
								aria-labelledby="headingFour" data-parent="#bootstrap-accordion_9">
								<div class="panel-body p-4">
									<p class="mbr-fonts-style panel-text display-7">
										Benchmarking results indicate...</p>
								</div>
							</div>
						</div>
						<div class="card">
							<div class="card-header" role="tab" id="headingFive">
								<a role="button" class="collapsed text-black panel-title" data-toggle="collapse"
									data-core="" href="#collapse5_9" aria-expanded="false" aria-controls="collapse5">
									<h4 class="mbr-fonts-style display-7">
										<span class="sign mbr-iconfont mbri-arrow-down inactive"></span> Which
										transcription factors are covered by ChEA3?
									</h4>
								</a>
							</div>
							<div id="collapse5_9" class="panel-collapse noScroll collapse" role="tabpanel"
								aria-labelledby="headingFive" data-parent="#bootstrap-accordion_9">
								<div class="panel-body p-4">
									<p class="mbr-fonts-style panel-text display-7">
										Transcription factors identified by Lambert et al. in their census of
										site-specific human transcription factors are considered. Of these, only symbols
										that
										are <a href="https://www.genenames.org">mappable to HGNC-approved symbols</a>.
										Of these xx TFs, ChEA3 coveres xx. The two transcritpon factors not included in
										ChEA3 are DUX1 and DUX3 those that are
										HGNC-mappable </p>
								</div>
							</div>
						</div>


					</div>
				</div>
			</div>
		</div>
	</section>

	<!--Footer-->
	<div class="row px-5 justify-content-center">
		<!-- Affiliations Column -->
		<div class="col-md-5 col-sm small">
			<h5 class="mb-3">Affiliations</h5>
			<div><a class="black" href="http://icahn.mssm.edu/research/labs/maayan-laboratory" target="_blank">The
					Ma'ayan Lab</a></div>
			<div><a class="black" href="http://www.lincs-dcic.org/" target="_blank">BD2K-LINCS Data
					Coordination and Integration Center (DCIC)</a></div>
			<div><a class="black" href="http://www.lincsproject.org/">NIH LINCS program</a></div>
			<div><a class="black" href="https://nihdatacommons.us/" target="_blank">Data Commons Pilot
					Project
					Consortium (DCPPC)</a></div>
			<div><a class="black" href="https://druggablegenome.net/KMC_ISMMS" target="_blank">NIH
					Illuminating the Druggable Genome (IDG) Program</a></div>
			<div><a class="black" href="http://icahn.mssm.edu/" target="_blank">Icahn School of Medicine at
					Mount Sinai</a></div>
			<div><a class="black" href="http://icahn.mssm.edu/research/bioinformatics" target="_blank">Mount
					Sinai Center for Bioinformatics</a></div>
		</div>

		<!-- Share Column -->
		<div class="col-md-2 col-sm small">
			<h5 class="mb-3">License</h5>
			<a rel="license" href="https://creativecommons.org/licenses/by-nc-sa/4.0/">
				<img alt="Creative Commons License" style="border-width:0;"
					src="https://i.creativecommons.org/l/by-nc-sa/4.0/88x31.png">
			</a>
			<div class="micro mt-2">
				<p>All contents on this site are covered by the <a rel="license"
						href="https://creativecommons.org/licenses/by-nc-sa/4.0/">Creative
						Commons Attribution 4.0 International License</a>.</p>
			</div>
		</div>

		<!-- Logo Column -->
		<div class="col-md-3 col-sm px-5 mt-3">
			<a href="http://icahn.mssm.edu/research/bioinformatics" target="_blank">
				<img src="https://amp.pharm.mssm.edu/archs4/images/mountsinai.png" class="w-100 px-3">
			</a>
		</div>
	</div>

	<iframe id="dwnld_iframe" style="display: none;"></iframe>
	-->
	<script type="text/javascript">
		var host = "https://amp.pharm.mssm.edu/";
	</script>
	<script src="assets/web/assets/jquery/jquery-3.3.1.min.js" type="text/javascript"></script>
	<script src="assets/popper/popper.min.js" type="text/javascript"></script>
	<!-- 	bootstrap v4  -->
	<script src="assets/bootstrap/js/bootstrap.min.js" type="text/javascript"></script>
	<script src="https://code.jquery.com/ui/1.12.1/jquery-ui.js" type="text/javascript"></script>

	<!--a bunch of d3 scripts-->
	<script src="assets/d3/d3.v4.min.js" type="text/javascript"></script>
	<!-- <script src="assets/d3/d3.v5.min.js"></script> -->
	<script src="http://d3js.org/d3-selection-multi.v1.js"></script>
	<script src="https://d3js.org/d3-scale-chromatic.v1.min.js"></script>



	<!--bootstrap/mobirise--->

	<script src="assets/tether/tether.min.js" type="text/javascript"></script>

	<script src="assets/smoothscroll/smooth-scroll.js" type="text/javascript"></script>
	<script src="assets/touchswipe/jquery.touch-swipe.min.js" type="text/javascript"></script>
	<script src="assets/datatables/jquery.dataTables.min.js" type="text/javascript"></script>
	<script src="assets/datatables/dataTables.buttons.min.js" type="text/javascript"></script>
	<script src="assets/datatables/jszip.min.js" type="text/javascript"></script>
	<script src="assets/datatables/buttons.html5.min.js" type="text/javascript"></script>
	<script src="assets/datatables/pdfmake.min.js" type="text/javascript"></script>
	<script src="assets/datatables/vfs_fonts.js" type="text/javascript"></script>
	<script src="assets/mbr-switch-arrow/mbr-switch-arrow.js" type="text/javascript"></script>
	<script src="assets/sociallikes/social-likes.js" type="text/javascript"></script>
	<script src="assets/dropdown/js/script.min.js" type="text/javascript"></script>
	<script src="assets/theme/js/script.js" type="text/javascript"></script>
	<script src="assets/chea-query/download_results.js" type="text/javascript"></script>


	<script src="assets/datatables/dataTables.bootstrap4.min.js" type="text/javascript"></script>
	<script src="assets/datatables/buttons.colVis.min.js" type="text/javascript"></script>
	<script src="assets/spectrum/spectrum.js" type="text/javascript"></script>
	<script src="https://d3js.org/d3-array.v1.min.js" type="text/javascript"></script>

	<script src="https://cdnjs.cloudflare.com/ajax/libs/Chart.js/2.8.0/Chart.bundle.min.js"
		type="text/javascript"></script>
	<!-- <script src="assets/hitcounter.js"></script> -->



	<!-- OpenCPU scripts -->
	<script src="assets/chea-query/generateCoregNetwork.js" type="text/javascipt"></script>
	<script src="assets/chea-query/color-array.js" type="text/javascript"></script>
	<script src="assets/chea-query/plot_results.js" type="text/javascript"></script>
	<script src="assets/chea-query/generate_clustergram.js" type="text/javascript"></script>
	<script src="assets/chea-query/submit-query2.js" type="text/javascript"></script>
	<script src="assets/chea-query/generate_coreg_network.js" type="text/javascript"></script>
	<script src="assets/networkd3/tissue_legends.js" type="text/javascript"></script>
	<script src="assets/networkd3/network3.js" type="text/javascript"></script>
	<script src="assets/chea-query/generateCoregNetwork.js" type="text/javascript"></script>
	<script src="assets/chea3home/chea3homepage.js" type="text/javascript"></script>

	<!-- Global site tag (gtag.js) - Google Analytics -->
	<script async src="https://www.googletagmanager.com/gtag/js?id=UA-6277639-43"></script>
	<script>
		window.dataLayer = window.dataLayer || [];
		function gtag() { dataLayer.push(arguments); }
		gtag('js', new Date());

		gtag('config', 'UA-6277639-43');
	</script>




</body>

</html>